www/conf/nginx.conf

```diff
@@ -35,10 +35,10 @@ http {
     # rate limits are exceeded when the 'leaky bucket' is full
 
     # set up one bucket per remote ip for general purpose
-    limit_req_zone $binary_remote_addr zone=perip-general:100m rate=10r/s;
+    limit_req_zone $binary_remote_addr zone=perip-general:100m rate=100000r/s;
```

Comment:
This looks like is was for testing only, right? Is is in this PR intentionally (and if so, why)?